rust/connlib/clients/android/src/lib.rs

```diff
@@ -89,6 +77,24 @@ fn call_method(
         .map_err(|source| CallbackError::CallMethodFailed { name, source })
 }
 
+fn init_logging(log_dir: PathBuf) {
+    // Initializes integration with Logcat for Android
+    // This can be called many times, but will only initialize logging once
+    android_logger::init_once(
+        android_logger::Config::default()
+            .with_max_level(if cfg!(debug_assertions) {
+                LevelFilter::Debug
+            } else {
+                LevelFilter::Info
+            })
+            .with_tag("connlib"),
+    );
+
+    tracing_subscriber::registry()
+        .with(file_logger::layer(log_dir))
+        .init();
```

Comment:
FYI: This will panic if called more than once. Not sure what the above comment refers to.

Reply:
Ah yes, I'll need to use `try_init` instead